packages/calcite-components/src/components/action-bar/action-bar.e2e.ts

```diff
@@ -165,10 +165,12 @@ describe("calcite-action-bar", () => {
       const page = await newE2EPage();
 
       await page.setContent(html`<calcite-action-bar expand-disabled></calcite-action-bar>`);
+      await page.waitForChanges();
 
       const buttonGroup = await page.find(`calcite-action-bar >>> .${CSS.actionGroupEnd}`);
 
-      expect(buttonGroup).toBeNull();
+      expect(buttonGroup.getAttribute("hidden")).toBe("");
```

Comment:
For consistency, can you check the prop instead?

Reply:
@jcfranco I can't do this I don't think?

![image](https://github.com/Esri/calcite-design-system/assets/1231455/94c1c0e0-2436-41a5-9485-73cf80cc4a65)

Reply:
Can't use getProperty as its not a component.

![image](https://github.com/Esri/calcite-design-system/assets/1231455/9bf7ef35-3f85-4f98-82ca-192707741e57)

Reply:
Maybe it's a doc thing, but we already do this in [`tab-title.e2e.ts`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden).

Reply:
yeah must be a doc thing.